app.py

```diff
@@ -755,6 +758,7 @@ def audit_report(election_id):
     
     report_writer.writerow(["Contest Name", contest.name])
     report_writer.writerow(["Number of Winners", contest.num_winners])
+    report_writer.writerow(["Number of Votes", contest.votes_allowed])
```

Comment:
I think we should change the CSV header to also be "Votes Allowed", for consistency.